tmt/cli.py

```diff
@@ -1750,9 +1750,21 @@ def status(
 # inference. See Context and ContextObjects above.
 @main.group(chain=True, invoke_without_command=True, cls=CustomGroup)  # type: ignore[arg-type]
 @pass_context
+@option(
+    '-l', '--last', is_flag=True, help='Clean everything of the last run.')
+@option(
+    '-i', '--id', 'id_', metavar="ID",
+    help='Run id (name or directory path) to clean everything of.')
+@option(
+    '-s', '--skip', type=str, default=None,
+    help='The clean phases to skip')
```

Comment:
How about accepting the option multiple times? That would be easier to process than encoding the multiple options into one use. See https://github.com/teemtee/tmt/blob/main/tmt/cli.py#L403 for an already existing option.

Plus, it would be nice to use `choices` and report `--skip=imags` as an error, i.e. it would need a list of possible steps one might ignore.

Reply:
@happz this seems to be resolved.